tokenserver/tests/assignment/test_sqlnode.py

```diff
@@ -28,7 +28,7 @@ def setUp(self):
         self.backend.add_service('sync-1.0', '{node}/1.0/{uid}')
         self.backend.add_service('sync-1.5', '{node}/1.5/{uid}')
         self.backend.add_service('queuey-1.0', '{node}/{service}/{uid}')
-        self.backend.add_node('sync-1.0', 'https://phx12', 100)
+        self.backend.add_node('sync-1.0', 'https://phx12', 100, nodeid=123)
```

Comment:
`SQLNodeAssignment.add_node` uses `node` as one of the parameters for the sql call to generate the user record. `node` is now a `NOT NULL` column, so we have to provide some value. I'm going to alter the function to make `node` listed param, since you're right, it should be noted that way and required.

Reply:
> node is now a NOT NULL column

The `id` column on the nodes table is supposed to be auto-increment in the same way that the `uid` column is on the `users` table, so is this stemming from the same auto-increment-handling problem as the `uid` case?  (Also note that `nodeid` and `node` are two different arguments, with the later being a non-nullable string column).